packages/ai/src/analysis.ts

```diff
@@ -273,7 +271,7 @@ Don't execute queries that modify data unless explicitly asked.`,
         };
       }
     },
-    message: queryMessage,
+    message: renderQueryMessageComponent,
```

Comment:
I need to update the openasistant/core to pass the persist messages when calling createAssistant(). I will create another PR to work on top of this PR.